src/components/HospitalPage/HospitalPageTitleRequestNarrative.tsx

```diff
@@ -0,0 +1,140 @@
+import { Box, Grid, LinearProgress, Typography } from "@mui/material";
+import { styled } from "@mui/system";
+
+const PurpleProgressBar = styled(LinearProgress)({
```

Comment:
A better naming strategy is to include how the ProgressBar is used, rather than how it looks.  Designers will change colors, but not purpose.

Reply:
Got it.